### PR TITLE
[frio] New Group button on group widget

### PR DESCRIPTION
**Pull request by anubis2814**
A simple plus sign takes you to the register page.
https://friendica.myportal.social/photos/anubis2814/image/3337502781647bf95ddc57b157281809

### CODE REVIEW

File: src/Content/GroupManager.php [hunk 1]
```diff
@@ -151,10 +151,12 @@ public static function widget(string $baseurl, int $uid, int $cid = 0)
 					'$title'	=> DI::l10n()->t('Groups'),
 					'$groups'	=> $entries,
 					'$link_desc'	=> DI::l10n()->t('External link to group'),
+					'$New_group_page' => 'register/',
```
Comment (nupplaphil) [inline src/Content/GroupManager.php]:
```suggestion
					'$new_group_page' => 'register/',
```

File: view/templates/widget/group_list.tpl [hunk 2]
```diff
@@ -16,9 +16,16 @@ function showHideGroupList() {
 	<h3>{{$title}}</h3>
 </span>
 <div id="group-list-sidebar" class="widget">
+<div id="sidebar-group-header" class="sidebar-widget-header">
 	<span class="fakelink" onclick="openCloseWidget('group-list-sidebar', 'group-list-sidebar-inflated');">
-		<h3 id="group-list">{{$title}}</h3>
+		<h3>{{$title}}</h3>
 	</span>
+	<a class="group-new-tool pull-right widget-action faded-icon" id="sidebar-new-group" href="{{$New_group_page}}" data-toggle="tooltip" title="{{$create_new_group}}">
```
Comment (nupplaphil) [inline view/templates/widget/group_list.tpl]:
```suggestion
	<a class="group-new-tool pull-right widget-action faded-icon" id="sidebar-new-group" href="{{$new_group_page}}" data-toggle="tooltip" title="{{$create_new_group}}">
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]
Just a minor snake case issue :-)

**Comment by MrPetovan** [comment]
Thank you for your ongoing efforts! I took the liberty of rebasing the branch onto the latest develop and combined several commits into one as well. Finally, I updated the main translation file by running `bin/run_gettext.sh` and committing the changes.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**